app/views/sign_up_sheet/_table_line.html.erb

```diff
@@ -1,4 +1,5 @@
-<td><%= topic.topic_identifier %></td>
+<td id='ident'><%= topic.topic_identifier %></td>
+<td><input type="checkbox" name="mySelectBox"></td>
```

Comment:
Change the checkbox name either selected-box or something, and change wherever is needed.